Utilities/StorageFactory/src/StorageAccount.cc

```diff
@@ -50,23 +86,49 @@ static std::string d2str(double d) {
   return t.str();
 }
 
+
+void StorageAccount::aggregateStatistics() {
+  std::set<std::string> protocols;
+  for (auto const &item : s_nameToToken) {
+    protocols.insert(item.first.second);
+  }
+  for (auto const &protocol : protocols) {
+    auto token = tokenForStorageClassName(protocol, OpenLabel::None);
```

Comment:
So is `OpenLabel::None` just meant to be used to denote an aggregation across all contexts? If so, maybe a more descriptive name would help clarify the logic?